### PR TITLE
Module BDII ready for puppet4

### PR DESCRIPTION
**Pull request by llobato**

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
-  $bdiihost             = hiera("bdii::params::bdiihost","somehost.example.com")
-  $bdiideletedelay      = hiera("bdii::params::deletedelay", "0")
-  $slapdthreads         = hiera("bdii::params::slapdthreads", "16")
-  $slapdloglevel        = hiera("bdii::params::slapdloglevel", "0")
-
+  $log_level            = hiera('bdii::params::log_level','DEBUG')
+  $port                 = hiera('bdii::params::port','2170')
+  $user                 = hiera('bdii::params::user','ldap')
+  $slapdconf            = hiera('bdii::params::slapdconf','/etc/bdii/bdii-slapd.conf')
+  $selinux              = hiera('bdii::params::selinux','True')
```
Comment (fschaer) [inline manifests/params.pp]: you're setting thee string to 'True' instead of using a bolean (still better than using the old unquoted True though...

Comment (llobato) [inline manifests/params.pp]: My bad. I have already changed :)

**Comment by fschaer** [comment]
Mmhh.. thanks.
Actually I just tried, and it seems that the selinux packages (emi2 ones) do not even exist in umd3 nor umd4, and I can’t find any other that would be named like that, so I believe this should just be erased as it’s a good example of a default config that just doesn’t work…

**Comment by andrea-manzi** [comment]
we merged #15 which contains also other fixes by Frederic

**Pull request closed without merging**